plugins/splunk-on-call/src/components/EntitySplunkOnCallCard.tsx

```diff
@@ -128,27 +199,32 @@ export const EntitySplunkOnCallCard = () => {
     return <Progress />;
   }
 
-  const Content = () => {
-    if (!team) {
-      return <MissingTeamAnnotation />;
-    }
-
-    if (!usersAndTeam?.foundTeam) {
-      return <InvalidTeamAnnotation teamName={team} />;
-    }
+  if (!usersAndTeams?.foundTeams || !usersAndTeams?.foundTeams.length) {
```

Comment:
If the used annotation is not associated with any teams, message the user accordingly...